src/dataviews/histogram-dataview-model.js

```diff
@@ -100,4 +109,16 @@ module.exports = DataviewModelBase.extend({
       }
     }, this);
   }
-});
+},
+
+  // Class props
+  {
+    ATTRS_NAMES: DataviewModelBase.ATTRS_NAMES.concat([
+      'column',
+      'column_type',
+      'bins',
+      'start',
```

Comment:
Yes, only `column` and `bins` are required for a normal histogram, `column_type`, `start` and `end` is required (set to `date` from DI) for the time-series use-case.